ports/unix/variants/minimal/mpconfigvariant.h

```diff
@@ -26,118 +26,44 @@
 
 // options to control how MicroPython is built
 
-// Prevent the rest of the default mpconfigport.h being used.
-#define MICROPY_UNIX_MINIMAL (1)
+#define MICROPY_CONFIG_ROM_LEVEL (MICROPY_CONFIG_ROM_LEVEL_MINIMUM)
```

Comment:
I would suggest renaming this variant from `minimal` to `minimum` to match the rom level name.

Reply:
It's the "minimum feature set" but the "minimal build".

Well, that was my logic at the time :) I'm happy to rename.

Reply:
I've thought / discussed in the past it'd be nice to have a variant that matches each of the rom levels, seems like this might be a good time to do it :-)

Reply:
Can do this in a separate PR.